vector.md

```diff
@@ -14,35 +14,35 @@ Also, dynamic arrays are faster than JavaScript's `Array` (else, all this would
 Just keep in mind that even if a `Int8Vector` really is faster, a `Float64Vector` won't give you an edge because this is often how an `Array` containing number would be optimized under the hood by most JavaScript engines.
 
 ```js
-var Vector = require('mnemonist/vector');
+const Vector = require('mnemonist/vector');
 ```
 
 ## Constructor
 
 The `Vector` takes a typed array class as first argument and an initial capacity or alternatively more complex options as second argument.
 
 ```js
-var vector = new Vector(ArrayClass, initialCapacity);
-
+const vector = new Vector(ArrayClass, initialCapacity);
+```
+```js
 // If you need to pass options such as a custom growth policy
-var vector = new Vector(ArrayClass, {
+const vector = new Vector(ArrayClass, {
   initialCapacity: 10,
   initialLength: 3,
-  policy: function(capacity) {
-    return Math.ceil(capacity * 2.5);
-  }
+  policy: (capacity) => Math.ceil(capacity * 2.5)  
 });
-
-// Subclasses for each of JS typed array also exists as a convenience
-var vector = new Vector.Int8Vector(initialCapacity);
-var vector = new Vector.Uint8Vector(initialCapacity);
-var vector = new Vector.Uint8Vector(initialCapacity);
-var vector = new Vector.Int16Vector(initialCapacity);
-var vector = new Vector.Uint16Vector(initialCapacity);
-var vector = new Vector.Int32Vector(initialCapacity);
-var vector = new Vector.Uint32Vector(initialCapacity);
-var vector = new Vector.Float32Vector(initialCapacity);
-var vector = new Vector.Float64Vector(initialCapacity);
+```
+Subclasses for each of JS typed array also exists as a convenience.
+```js
+vector = new Vector.Int8Vector(initialCapacity);
+vector = new Vector.Uint8Vector(initialCapacity);
+vector = new Vector.Uint8Vector(initialCapacity);
+vector = new Vector.Int16Vector(initialCapacity);
+vector = new Vector.Uint16Vector(initialCapacity);
+vector = new Vector.Int32Vector(initialCapacity);
+vector = new Vector.Uint32Vector(initialCapacity);
+vector = new Vector.Float32Vector(initialCapacity);
+vector = new Vector.Float64Vector(initialCapacity);
```

Comment:
const is missing on each line or `let vector` instead of `const vector` before.

Reply:
I did not want to introduce `let` because every other example uses `const`. I also did not want to split the example as this would draw too much attention to the repetitive convenience subclass.
I would go with different variable names or one generic example:
```js
const { Int32Vector } = Vector; 
// available for every JS typed array:  Int8Vector, Uint8Vector, ..., Float64Vector

const initialCapacity = 100;
const vector = new Int32Vector(initialCapacity); 
```

Also I believe the convenience is not convenient enough, I would happily use `new Vector(Int8Array, 100)` over `new Vector.Int8Vector(100)`. Maybe we can get rid of the whole example.